frontend/src/components/pages/TournamentRegistrations/TournamentRegistrations.tsx

```diff
@@ -133,33 +162,54 @@ const RegistrationsCard = styled(Card)({
 });
 const PlayersList = (props: {
   players: PlayerProfileDto[];
```

Comment:
Nie wiem czy ładniej nie byłoby zrobić, że tutaj players to `players: (PlayerProfileDto & {isRegistered: boolean})[]`
i to łączyć przed podaniem do komponentu.

Reply:
coś spróbuję pokombinować

Reply:
@Szambelan Ale zrob najpierw tak, zeby dzialalo :)

Reply:
no działa przecież, chyba, że masz na myśli nieprzechodzące testy

Reply:
Tak, tak - ze na razie mozesz to zsotawic, ten context.